### PR TITLE
[RFR] Add draw management & test

### PR DESCRIPTION
**Pull request by HALLERPierre**
[Trello](https://trello.com/c/lbLAbtnp/9-as-fred-if-i-align-5-thomass-shapes-i-lost-even-if-i-align-5-of-my-shapes-025)

# Done

- [x] Add current player to check win functions
- [x] Return the non-playing player if the board detect 2 winners
- [x] Add test

# Result
draw

### CODE REVIEW

File: src/game/success.py [hunk 1]
```diff
@@ -20,23 +20,22 @@ def check_diag(board, inverted=False):
...
 
 
-def check_rows(board):
-    sum_row = 0
+def check_rows(board, player=0):
+    winner=0
     for row in board:
-        winner = get_winner(sum(row))
-        if (winner != 0):
+        winner = get_winner(sum(row)) if get_winner(sum(row)) != 0 else winner
```
Comment (zyhou) [inline src/game/success.py]: add variable for `get_winner` result

File: src/game/success.py [hunk 2]
```diff
@@ -20,23 +20,22 @@ def check_diag(board, inverted=False):
...
+        if winner < 0 and player == 1 or winner > 0 and player == -1:
```
Comment (zyhou) [inline src/game/success.py]: No need to add parenthesis ?

**Review by zyhou — APPROVED** [review]
+1 about the readme. Don't forget to add gif :)

File: src/game/success.py (hunk 2 above)
Comment (Luwangel) [reply src/game/success.py]: It's not so useful imo

Comment (HALLERPierre) [reply src/game/success.py]: Added for better readability

**Review by zyhou — APPROVED** [review]

**Pull request merged**